Assets/MRTK/Core/Definitions/Devices/ArticulatedHandDefinition.cs

```diff
@@ -120,8 +118,14 @@ public MixedRealityInteractionMapping[] DefaultInteractions
             new MixedRealityInputActionMapping("Teleport Pose", AxisType.DualAxis, DeviceInputType.ThumbStick),
         };
 
+
+        // Internal calculation of what the HandRay should be
+        // Useful as a fallback for hand ray data
+        protected virtual IHandRay HandRay { get; } = new HandRay();
+
         /// <summary>
         /// Calculates whether the current pose allows for pointing/distant interactions.
+        /// Equivalent to the HandRay's ShouldShowRay implementation (how do I hotlink the code reference in here Kurtis)
```

Comment:
> (how do I hotlink the code reference in here Kurtis)

🤣 you should be able to use the [see or seealso](https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/language-specification/documentation-comments#d314-see) tags

Reply:
Done